src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+spring.application.name=ilimPaymentHandlerServ
+# Server Configuration
+server.port=8080
```

Comment:
can you changer the port? because we assigned 8080 to IlimDeliveryServ in the current one ec2 instance deployment strategy.

Reply:
addressed thanks